### PR TITLE
Add openacademie startup

### PR DESCRIPTION
**Pull request by MattiSG**
@ismael-hery Please review  ;)
Missing contact & start date in particular.
Examples here: https://github.com/sgmap/sgmap.github.io/blob/master/_startups/embauche.md https://github.com/sgmap/sgmap.github.io/blob/master/_startups/mes-aides.md

### CODE REVIEW

**Comment by MattiSG** [comment]
Ok ?

**Comment by Ismael-Hery** [comment]
Ok en prenant en compte les rqes.

File: _startups/openacademie.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Open Académie
+mission: Des outils et des méthodes au service de la modernisation de l’action publique dans l’Éducation nationale.
+domain: openacademie.sgmap.fr
+repository: https://github.com/sgmap/open-academie
+status: consolidation
+contact: pierre@pezziardi.net
```
Comment (MattiSG) [inline _startups/openacademie.md]: @ppezziardi ok ?

Comment (ppezziardi) [inline _startups/openacademie.md]: contact: pierre.de-maulmont@ac-paris.fr

**Pull request merged**